packages/react-native/ReactAndroid/src/test/java/com/facebook/react/bridge/interop/InteropModuleRegistryTest.kt

```diff
@@ -82,7 +80,7 @@ class InteropModuleRegistryTest {
 
     val interopModule = underTest.getInteropModule(RCTEventEmitter::class.java)
 
-    assertTrue(interopModule is FakeRCTEventEmitter)
+    assertThat(interopModule is FakeRCTEventEmitter).isTrue()
```

Comment:
For all the instance checks can you change to:

```
assertThat(interopModule).isInstanceOf(FakeRCTEventEmitter::class.java)
```

Reply:
Done. https://github.com/facebook/react-native/pull/45623/commits/7483acc18ff6349fb675db83a1bce7b1818d8648